src/lib/dialog/dialog-config.ts

```diff
@@ -32,6 +32,9 @@ export class MdDialogConfig {
   /** Position overrides. */
   position?: DialogPosition;
 
+  /** Classes to add to the dialog container. */
+  containerClass?: string | string[] = '';
```

Comment:
I'll undo this since we plan to move away from using dialog, I can hack around it for now